core/commands/storage.go

```diff
@@ -232,7 +232,7 @@ Use status command to check for completion:
 			return err
 		}
 
-		go retryMonitor(storage.NewGoContext(req.Context), api, output.ss, n, output.ssID, output.testFlag,
+		go retryMonitor(output.ss.RetryMonitorCtx , api, output.ss, n, output.ssID, output.testFlag,
```

Comment:
There is more than one place you use `go retryMonitor(...)`, also since `output.ss` is passed into `retryMonitor`, you can use it directly inside it, instead of passing in.